### ISSUE TITLE
Game not working on Kickstart 1.2 / 1.3 from floppy

### ISSUE THREAD

**Issue by rveilleux** [Last-Minute-Creations/germz#1]
I was not able to boot the ADF on WINUAE when using a kickstart 1.2 or 1.3: it loads for a little while but seems to stay stuck as soon as the game attempts to open a file. Compiling from source yields the same result as trying the prebuilt ADF from https://lastminutecreations.itch.io/germz.
From Kickstart 2.00 and upward, it works. And on older kickstarts, it works when launched from HDD too. So the problem seems to be a mixed of trackdisk loading combined with older kickstart rom. 
Are you experiencing the same issue? I'm seeing a similar problem on my real Amiga 2000 with a GOTEK drive too when trying my own code. I'll try Germz to see if the same situation happens.

**Comment by tehKaiN** [comment]
Yeah, this is known issue. KS1.3 handles fdd in odd way (e.g. uses blitter for disk operations). It's really a game development library (ACE) problem. I don't know if it will be fixed in the first full GermZ release, but I'll try to fix it ;)
